packages/api/src/controllers/helpers.ts

```diff
@@ -234,6 +237,35 @@ export async function getS3PresignedUrl(os: ObjectStore, objectKey: string) {
   return getSignedUrl(s3, putCommand, { expiresIn });
 }
 
+export async function generateRequesterId(req: Request, playbackId: string) {
+  const origin =
```

Comment:
I would use a different name than `origin`, since this is already it is already [well-defined in HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) and is not what it means here. Why not just call this `ip`?